extensions/amp-next-page/1.0/service.js

```diff
@@ -276,13 +273,11 @@ export class NextPageService {
     return Promise.all(
       previousPages
         .filter(page => {
-          const shouldHide =
-            page.relativePos === ViewportRelativePos.LEAVING_VIEWPORT ||
-            page.relativePos === ViewportRelativePos.OUTSIDE_VIEWPORT;
-          return shouldHide;
+          // Pages that are outside of the viewport should be hidden
+          return page.relativePos === ViewportRelativePos.OUTSIDE_VIEWPORT;
```

Comment:
Same as above, why isn't it breaking to remove the `LEAVING_VIEWPORT` condition?